apps/web-twig-demo/src/Controller/ComponentsController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ComponentsController extends AbstractController
+{
+    #[Route('/components/{component}', name: 'component_view')]
+    public function show(string $component): Response
+    {
+        return $this->render(sprintf('@components/%s/%s.stories.twig', $component, $component));
+    }
+
+    /**
+     * @return Array<string>
+     */
+    private function getWebTwigComponents(): array
```

Comment:
This one should be in a `service` to prevent duplicating it in each controller.

Reply:
Copy&paste bug. Removed.